Ekim/documentation suggestions (#202)
CitrineInformatics/citrine-python: docs/source/workflows/performance_workflows.rst

By comparing the model's predictions to the true values, we can compute several performance metrics that provide information about model quality.

- Root-mean squared error (RMSE): the square root of the average of the squared prediction error

> - Non-dimensional model error (NDME): The RMSE, normalized by the RMSE of a trivial model that always predicts the mean value of the data (== standard deviation of the output variable).

Reviewer: ```suggestion
- Non-dimensional model error (NDME): The RMSE, normalized by the standard deviation of the output variable (which is the RMSE of a trivial model that always predicts the mean value of the data).
```
Your addition made me think that we should just lead with the standard deviation. Feel free to reject if you think this is less legible

- Standard residual: The RMSE of the standardized error (the prediction error divided by the prediction uncertainty)
- Standard confidence: The fraction of predictions for which the prediction error is less than the prediction uncertainty

Performance metrics included in the results depend on whether the response is numeric or categorical.
For numeric responses, performance metrics include root mean squared error (RMSE), non-dimensiona model error (NDME), standard residual and standard confidence.

-  RMSE is a useful and popular statistical metric for model quality.
   Lower RMSE means the model is more accurate.
-  NDME is the ratio between RMSE and standard deviation of the output variable.
   NDME is a useful non-dimensional model quality metric.
   A value of NDE = 0 is a perfect model. If NDE = 1, then the model is uninformative.

>    An acceptable NDE depends on how the model is used.

Reviewer 2: ```suggestion
   An acceptable NDME depends on how the model is used.
```

- Standard residual: The RMSE of the standardized error (the prediction error divided by the prediction uncertainty)
- Standard confidence: The fraction of predictions for which the prediction error is less than the prediction uncertainty

Performance metrics included in the results depend on whether the response is numeric or categorical.
For numeric responses, performance metrics include root mean squared error (RMSE), non-dimensiona model error (NDME), standard residual and standard confidence.

-  RMSE is a useful and popular statistical metric for model quality.
   Lower RMSE means the model is more accurate.
-  NDME is the ratio between RMSE and standard deviation of the output variable.
   NDME is a useful non-dimensional model quality metric.
   A value of NDME = 0 is a perfect model. If NDME = 1, then the model is uninformative.
   An acceptable NDME depends on how the model is used.
   Generally, NDME > 0.9 indicates a model with very high error.
   If 0.9 > NDME > 0.6, this model is typically a good candidate for a design workflow.

>    Lower values of NDME indicate increasingly accurate models.

Reviewer 2: Here would be a nice place to explain what a lower value means as a candidate for a design workflow, continuing the thought from the previous sentence.

Author: Changed! lmk what you think in the new update

   If AUC=1.0, all samples are classified correctly.
   If AUC=0.5, the model cannot distinguish between the two categories.
   If AUC=0.0, all samples are classified incorrectly.

> -  Support-weighted F1 score is calculated from averaged precision and recall of the model, weighted by the in-class fraction of true positives according to the formula ``2.0 * precision * recall / (precision + recall) * fraction_true_positives`` applied to each class.

Reviewer: ```suggestion
-  Support-weighted F1 score is calculated from averaged precision and recall of the model, weighted by the in-class fraction of true positives according to the formula ``2.0 * precision * recall / (precision + recall) * fraction_true_positives`` summed over each class.
```
